vyper/signatures/interface.py

```diff
@@ -169,12 +166,11 @@ def render_decorator(sig):
         if idx == 0:
             out += "\n# Functions\n"
         if not func.private and func.name != '__init__':
-            out += "{decorator}def {name}({args}){ret}:\n    pass\n".format(
-                decorator=render_decorator(func),
-                name=func.name,
-                args=", ".join([arg.name + ": " + str(arg.typ) for arg in func.args]),
-                ret=render_return(func)
-            )
+            args = ", ".join([arg.name + ": " + str(arg.typ) for arg in func.args])
+            out += (
+                    f"{render_decorator(func)}def {func.name}({args}){render_return(func)}:\n"
```

Comment:
Combined with above and below
```suggestion
            out += f"""
{render_decorator(func)}
def {func.name}({args}){render_return(func)}:
    pass
"""
```